### PR TITLE
Manual deployment instructions

### PR DESCRIPTION
**Pull request by dkelosky**
This PR only introduces doc to cover the manual steps involved in running the service on z/OS via java command in USS or as a batch job.

In a future PR, I hope to introduce some automation to render JCL and deployment of artifacts through npm and zowe CLI.

### CODE REVIEW

**Comment by plavjanik** [comment]
Hi Dan, I am still using https://github.com/plavjanik/sample-spring-boot-api-service/ until CircleCI or another CI build will be done for zowe/sample-spring-boot-api-service. Your instructions look great and superior to the older ones - https://github.com/plavjanik/sample-spring-boot-api-service/blob/master/docs/zos-deployment.md.

File: docs/deployment-instructions.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    - `zowe files upload ftu "<path_to_local_file>/sample-service.jar" "/u/ibmuser/samplapi/jars/sample-service.jar" --binary`
+
+#### Deploy the Sample Service Configuration YAML
+
+1. Create a directory for the `application.yml`
+   - `mkdir /u/ibmuser/samplapi/config`
+
+2. Upload the `config/local/application.yml` as a binary artifact:
+
+`zowe files upload ftu "config/local/application.yml" "/u/ibmuser/samplapi/config/local/application.yml" --binary`
```
Comment (plavjanik) [inline docs/deployment-instructions.md]: I would remove `local` from `/u/ibmuser/samplapi/config/local/application.yml`. The `local` meant configuration for running on a *local* workstation.

**Comment by plavjanik** [comment]
Dan, I have synced the master of `zowe` and `plavjanik` repositories. Thank you again for writing the instructions. If you see something good in https://github.com/zowe/sample-spring-boot-api-service/blob/master/docs/zos-deployment.md, please add it to your instructions and remove `docs/zos-deployment.md`.

**Comment by dkelosky** [comment]
Oops, sorry for the troubles.  I'll sort out the differences tomorrow in the proper places :smile:

**Comment by plavjanik** [comment]
Sorry for the troubles. I should make it clear where we will make changes. We need to CircleCI, Azure Pipelines, or create Jenkinsfile for wash.zowe.org.

**Comment by dkelosky** [comment]
Anything but Jenkins 😎

**Comment by plavjanik** [comment]
I am closing it since it has been pushed from plavjanik org.

**Pull request closed without merging**